src/helpers/parser/listItemParser.js

```diff
@@ -180,6 +180,7 @@ const parseCategories = (data, skipLastDivider, routeName = 'Category') => {
     pointsOfInterestCount: category.pointsOfInterestCount,
     toursCount: category.toursCount,
     routeName,
+    parent: category.parent ? true : false,
```

Comment:
is this especially needed, or would `parent: category.parent` already work? the value of parent could be checked elsewhere as well, instead of having it `true` or `false` i guess.
on the other hand, if we would need to access data of the parent, these data were gone and exchanged with the `true`, right?